utilities/after_success.sh

```diff
@@ -34,9 +34,9 @@ if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" ]; then
 
             cd ..
             utilities/update_docs_version.sh # Update version in READMEs
-            mvn clean deploy --settings ~/.m2/settings.xml -P sign-deploy
+            mvn clean deploy --settings ~/.m2/settings.xml -P sign-deploy,release
```

Comment:
I completely forgot how we got to the sign-deploy profile (who defines it and is order matters?) I assume you tested it?